src/KiD_driver.jl

```diff
@@ -23,59 +27,46 @@ n_elem = 256
 t_ini = 0.0
 t_end = 10.0 * 60
 
-# Updraft momentum flux terms and initial conditions
+# Updraft momentum flux terms, surface terms and initial conditions
 w1 = 2 # m/s * kg/m3
 t1 = 600 # s
 w_params = (w1 = w1, t1 = t1)
+q_surf = 0.016
+ρw0 = 0.0
 
-domain = Domains.IntervalDomain(
-    Geometry.ZPoint{FT}(z_min),
-    Geometry.ZPoint{FT}(z_max),
-    boundary_names = (:bottom, :top),
-)
-mesh = Meshes.IntervalMesh(domain, nelems = n_elem)
+# initialize the timestepping struct
+TS = TimeStepping(FT(Δt), FT(Δt_output), FT(t_end))
+
+# create the coordinates,
+space, face_space = make_function_space(z_min, z_max, n_elem)
```

Comment:
Just curious the rationale for doing it this way? I notice we still do it the old way in run_tests.jl

Reply:
I ditched the `run_tests.jl` for now. We should not be duplicating things. Hopefully we can add better tests - like comparing with PySDM soon.